docs/content/common-subs/kubestellar-apply-apache.md

```diff
@@ -0,0 +1,111 @@
+<!--kubestellar-apply-apache-start-->
+<span class="Space-Bd-BT">KUBESTELLAR</span>'s helm chart automatically creates a Workload Management
+Workspace (WMW) for you to store kubernetes workload descriptions and <span class="Space-Bd-BT">KUBESTELLAR</span> control objects in. The automatically created WMW is at `root:wmw1`.
+
+Create an EdgePlacement control object to direct where your workload runs using the 'location-group=edge' label selector. This label selector's value ensures your workload is directed to both clusters, as they were labeled with 'location-group=edge' when you issued the 'kubestellar prep-for-cluster' command above.
+
+In the `root:wmw1` workspace create the following `EdgePlacement` object: 
+```shell linenums="1" hl_lines="10 11 16 21"
+export KUBECONFIG=ks-core.kubeconfig
+kubectl ws root:wmw1
+
+kubectl apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v2alpha1
+kind: EdgePlacement
+metadata:
+  name: my-first-edge-placement
+spec:
+  locationSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - apiGroup: ""
+    resources: [ configmaps ]
+    namespaceSelectors:
+    - matchLabels: {"common":"sure-is"}
```

Comment:
With release 0.8 and 0.9 you can replace this namespace label selection with simply `namespaces: [ my-namespace ]`.